src/components/forum/ForumMessage.vue

```diff
@@ -197,7 +197,7 @@ export default defineComponent({
       return (value / 1_000_000).toFixed(0)
     },
     markedMessage (text: string) {
-      const html = DOMPurify.sanitize(marked(text))
+      const html = DOMPurify.sanitize(marked(text, { renderer: renderer }), { ADD_ATTR: ['target'] })
```

Comment:
Given that these are always included together, I'd make a a "RenderUserInput" function or something and import that instead which does all this.